### DISCUSSION TITLE
username and password

### DISCUSSION THREAD

**Discussion by armordturtle** [community/community#44228] [New to GitHub]
### Select Topic Area

Question

### Body

Hello, 
Currently, I am taking my first programming class. All of my assignments are provided by the HTTPS. So, every time I commit my Project, eclipse asks my username and password. I enter my Git username and password, but it doesn't work. I have been looking for the answers online, but I ended up getting confused and don't know how to move forward. If you have time, could you give me any suggestions please?

**Comment by alexeytatarchuk** [comment]
You can try using GitHub Desktop: https://desktop.github.com/

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @armordturtle ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
